content/applications/sales/sales/products_prices/products/variants.rst

```diff
@@ -25,201 +25,147 @@ colors). Each one of these products has its own inventory, sales, etc.
-It is also accessible from the Purchase and inventory modules.
-
-Now, click on the product you wish to add variants to.
-
-In the product page, a new tab called Variants has appeared. The number
-in purple written on top is the number of variants this product
-currently has. To add new variants, click on the tile. In the new
-window, click on **Create**.
+Head over to the Products page by clicking :menuselection:`Sales → Products --> Products`, and
+then, either select a product to add variants to or create a new one by tapping **Create**.
 
-In **Attributes**, click on the rolldown menu and select the type of
-variance you wish to add. If the variant does not yet exist, you can
-create it on the fly by clicking on Create and edit…
+On the product page, a new tab called Variants has appeared. The smart button at the top of the
+template indicates the number of currently configured variants on this product.
 
-.. image:: media/variant02.png
+.. image:: variants/variant-smart-button.png
    :align: center
+   :alt: Variants smart button
 
-In the Attributes window, the **Value** field is the description of the
-attribute such as Green, Plastic or 32GB. The **Attribute** field is the
-type of variant such as Color, Material or Memory.
+To add a new variant, click on the **Variants Tab**, then on **Add Any Line** to add any
+attributes and values to create new product options for your customers.
 
-.. image:: media/variant03.png
-   :align: center
-
-You can add a cost for the variant on the fly by adding it in the
-**Attribute Price Extra** field, or choose to modify it later. Click on
-**Save**.
-
-You can also add a different barcode and internal reference to the
-variant.
-
-When you have entered all the specifications of the variant, click on
-**Save**.
+.. tip::
+   The order of Attributes on the Attributes page dictates how they appear on the Product
+   Configurator, PoS dashboard or eCommerce pages.
 
-Managing Product Variants
-=========================
+Similar creation processes are accessible through the Purchase, Inventory, and eCommerce
+applications.
 
-Introduction
-------------
+Managing product exclusions
+---------------------------
 
-The examples below are all based on this product template that has two
-variant attributes :
+The examples below are all based on this product template that has two variant attributes :
 
--  T-Shirt B&C
+-  T-Shirt
 
    -  Color: Red, Blue, White
 
    -  Size: S, M, L, XL, XXL
 
-Managing combination possibilities
-----------------------------------
-
-By default, with the above product template, you get 15 different
-products (3 colors, 5 sizes). If the XXL size only exists for red and
-blue t-shirts, you can deactivate the white product variant.
+With the above product template, you have 15 different t-shirt products in 3 different colors and
+5 different sizes. If the red and blue t-shirts are the only shirts available in XXL, you can
```

Comment:
```suggestion
With the above product template, you have 15 different t-shirt products in three different colors and
five different sizes. If the red and blue t-shirts are the only shirts available in XXL, you can
```
Most of the time we should spell out numbers from one to ten (unless it's a calculation or an amount of money)